CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.7.1
+  - Fix: do not force binary encoding on template [#128](https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/128)
```

Comment:
Our changelog should hold the _WHY_, to empower people who are encountering problems to find which releases fix those problems.

```suggestion
  - Fix: solves an issue where non-ascii unicode values in a template were not handled correctly [#128](https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/128)
```